pkg/api/norman/store/cluster/cluster_store.go

```diff
@@ -1099,3 +1099,10 @@ func canUpgrade(nodes []*v3.Node, upgradeStrategy *rketypes.NodeUpgradeStrategy)
 	}
 	return nil
 }
+
+func comparePrivateRegistryUrls(newConfig map[string]interface{}, oldConfig map[string]interface{}) bool {
```

Comment:
Is this function name based on another one in this file? The function is called `comparePrivateRegistryUrls` but the return is a boolean, this probably needs an expression so that the boolean confirms or denies the expression

Did we also check if `replaceWithOldSecretIfNotExists` cannot be used?

Reply:
Sorry about the bad naming, I will fix that if we use the new method.

I haven't checked `replaceWithOldSecretIfNotExists`, but it looks like that is explicitly expecting a cloudProvider at the moment.  The idea behind a new method was to keep the logic as close as possible to what was there already.